### DISCUSSION TITLE
Sorting of messages by Newest first

### DISCUSSION THREAD

**Discussion by rackom** [provectus/kafka-ui#3354] [Q&A]
Sorting by `Newest first` inside topic sorts messages by newest first, but only in each partition. Is it a bug or feature? I would expect that `Newest first` works across all partitions. This can be somehow overcome by using `Live Mode` that sorts messages as they are received but for historical searches it is not usable.

**Comment by Haarolean** [comment]
Hey, I'll convert this to an issue.
